bindings/matrix-sdk-crypto-js/src/requests.rs

```diff
@@ -217,21 +227,39 @@ pub struct RoomMessageRequest {
     #[wasm_bindgen(readonly)]
     pub id: Option<JsString>,
 
-    /// A JSON-encoded object of form:
+    /// A string representing the room to send the event to.
+    #[wasm_bindgen(readonly)]
+    pub room_id: JsString,
+
+    /// A string representing the transaction ID for this event.
     ///
-    /// ```json
-    /// {"room_id": …, "txn_id": …, "content": …}
-    /// ```
+    /// Clients should generate an ID unique across requests with the same
+    /// access token; it will be used by the server to ensure idempotency of
+    /// requests.
     #[wasm_bindgen(readonly)]
-    pub body: JsString,
+    pub txn_id: JsString,
+
+    /// A string representing the type of even from the message's content.
+    #[wasm_bindgen(readonly)]
+    pub event_type: JsString,
+
+    /// A JSON-encoded string containing the message's content.
+    #[wasm_bindgen(readonly)]
+    pub content: JsString,
```

Comment:
it's a bit odd that `RoomMessageRequest` ends up with a `content` whereas all of the other request structs have a `body`, but not the end of the world.

Reply:
Oh yeah, I can rename to `body`. Thanks.